packages.yml

```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/pinterest_source
-  version: [">=0.5.0", "<0.6.0"]
+# - package: fivetran/pinterest_source
+#   version: [">=0.5.0", "<0.6.0"]
+  - git: https://github.com/fivetran/dbt_pinterest_source.git
```

Comment:
Why not merge and release pinterest source first?

Reply:
Good call out! We will merge pinterest source first. However, we usually do this to ensure CircleCi successfully runs. This will be switched before merge.